e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -442,7 +442,7 @@ void compilationFailsOutdatedSimulationSimDatasetId() throws IOException {
       final var message = exception.getMessage().split("\"message\":\"")[1].split("\"}]")[0];
       // Hasura strips the cause message ("Assumption falsified -- mission model for existing plan does not exist")
```

Comment:
Minor: can this comment be updated to reflect the error message now actually being stripped?